### PR TITLE
generating text output for many charts

### PR DESCRIPTION
**Pull request by voj**

### CODE REVIEW

File: src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java [hunk 1]
```diff
@@ -2632,9 +2632,10 @@ public static File plotRuptureHistogram(File outputDir, String prefix,
 		gp.getChartPanel().setSize(800, 600);
 		File pngFile = new File(outputDir, prefix+".png");
 		File pdfFile = new File(outputDir, prefix+".pdf");
+		File txtFile = new File(outputDir, prefix+".txt");
 		gp.saveAsPNG(pngFile.getAbsolutePath());
 		gp.saveAsPDF(pdfFile.getAbsolutePath());
-		
+        gp.saveAsTXT(txtFile.getAbsolutePath());
```
Comment (chrisbc) [inline src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java]: @voj this indentation looks off in github

**Review by chrisbc — CHANGES_REQUESTED** [review]
@voj Can you also add this mod to the class NZSHM22_InversionRateDiagnosticsPlot (if it'snot threre already??

File: src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java (hunk 1 above)
Comment (voj) [reply src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java]: the original lines have tabs :/

Comment (voj) [reply src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/RupSetDiagnosticsPageGen.java:2638]: tabs :-(

**Comment by voj** [comment]
> 
> 
> @voj Can you also add this mod to the class NZSHM22_InversionRateDiagnosticsPlot (if it's not threre already??

already merged into our master!

**Comment by voj** [comment]
@chrisbc if you're happy with this I'll create an upstream PR

**Comment by voj** [comment]
merged into upstream

**Pull request closed without merging**